jdk/make/closed/autoconf/custom-hook.m4

```diff
@@ -380,9 +380,6 @@ AC_DEFUN([OPENJ9_CHECK_NASM_VERSION],
 
 AC_DEFUN([OPENJDK_VERSION_DETAILS],
 [
-  # Source the closed version numbers
-  . $SRC_ROOT/jdk/make/closed/autoconf/openj9ext-version-numbers
```

Comment:
Why does this need to move? Isn't `CUSTOM_EARLY_HOOK` early enough?

Reply:
Correct, `CUSTOM_EARLY_HOOK` is not early enough.

Reply:
Besides not being early enough, not doing it here is just problematic. The values are contained in the generated spec.gmk file, and if it's not done here spec.gmk will have the wrong values, which could get used when building various parts of the JDK. i.e. even if setting COMPANY_NAME later is used for some part of the OpenJ9 build, there are likely other parts using the wrong value.

Reply:
CUSTOM_EARLY_HOOK is run before JDKOPT_SETUP_JDK_VERSION_NUMBERS, but JDKOPT_SETUP_JDK_VERSION_NUMBERS runs `$AUTOCONF_DIR/version-numbers` which would overwrite any values set earlier.

Reply:
I've done some editing of the above comments, best to read the latest versions.